selfdrive/navd/navd.py

```diff
@@ -85,6 +86,27 @@ def update_location(self):
     if self.localizer_valid:
       self.last_bearing = math.degrees(location.calibratedOrientationNED.value[2])
       self.last_position = Coordinate(location.positionGeodetic.value[0], location.positionGeodetic.value[1])
+      if self.timezone is None:
```

Comment:
we want to update more than this, imagine you cross a time zone boundary on a road trip.

@sshane you've hit this a few times on road trips. what do you think a reasonable policy for this is?

Reply:
I can't see option to get notified of timezone changes, or when they happen on a route. I suppose every minute with a backoff would be fine.

Reply:
hmm i had this in mind when first working on this but switched to using params since how often does timezone actually change when it's being used, it also matched the previous implementation of `timezonefinder`. But you brought up a good point. I'll switch back to adding `timezone` into the `NavRoute` packet.